ansible_collections/arista/avd/roles/eos_designs/templates/documentation/fabric-p2p-links.j2

```diff
@@ -2,34 +2,36 @@ Type,Node,Node Interface,Leaf IP Address,Peer Type,Peer Node,Peer Interface,Peer
 {% set interfaces_done = [] %}
 {% for node in groups[fabric_name] | arista.avd.natural_sort %}
 {%     set node_hostvars = hostvars[node] %}
-{%     if node_hostvars.type | arista.avd.default('undefined') in node_type_keys | dict2items | map(attribute='value.type') | list %}
-{%         for ethernet_interface in node_hostvars.ethernet_interfaces | arista.avd.natural_sort %}
-{%             if node_hostvars.ethernet_interfaces[ethernet_interface].type is arista.avd.defined('routed') %}
-{%                 if node_hostvars.ethernet_interfaces[ethernet_interface].peer_type | arista.avd.default('undefined') in node_type_keys | dict2items | map(attribute='value.type') | list %}
-{%                     do interfaces_done.append(node ~ "," ~ ethernet_interface) %}
-{%                     set peer = node_hostvars.ethernet_interfaces[ethernet_interface].peer %}
-{%                     set peer_interface = node_hostvars.ethernet_interfaces[ethernet_interface].peer_interface %}
-{%                     if peer is arista.avd.defined and peer_interface is arista.avd.defined and peer ~ "," ~ peer_interface not in interfaces_done %}
-{%                         set csv_line = [] %}
-{%                         do csv_line.append(node_hostvars.type) %}
-{%                         do csv_line.append(node) %}
-{%                         do csv_line.append(ethernet_interface) %}
-{%                         do csv_line.append(node_hostvars.ethernet_interfaces[ethernet_interface].ip_address | arista.avd.default("")) %}
-{%                         do csv_line.append(node_hostvars.ethernet_interfaces[ethernet_interface].peer_type) %}
-{%                         do csv_line.append(peer) %}
-{%                         do csv_line.append(peer_interface) %}
-{%                         set peer_ip_address = "" %}
-{%                         set peer_hostvars = hostvars[peer] | arista.avd.default %}
-{%                         if peer_hostvars.ethernet_interfaces is arista.avd.defined %}
-{%                             if peer_hostvars.ethernet_interfaces[peer_interface] is arista.avd.defined %}
-{%                                 set peer_ip_address = peer_hostvars.ethernet_interfaces[peer_interface].ip_address | arista.avd.default("") %}
+{%     for node_type_key in node_type_keys | arista.avd.convert_dicts('name') %}
+{%         if node_hostvars.type | arista.avd.default('undefined') in node_type_key.name %}
```

Comment:
This goes with the previsou comment - here we are checking that the type is "in" a string rather than in a list of string as it was before.

```suggestion
{%         if node_hostvars.type | arista.avd.default('undefined') in node_type_keys | arista.avd.convert_dicts('name') | map(attribute='name') %}
```